### DISCUSSION TITLE
Open tab with DataObject

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#9631] [Q&A / Support]
In the admin area. How to make a button that will open tab with DateObject?

**Comment by esisl** [comment]
Developers!
Please!
Tell me how to open a tab for editing an object by its identifier in the admin panel (in edit-mode) with a js-script?

I can of course do this:
`var ddd = parent.document.getElementById('pimcoretreeview-1136-record-2489');
    ddd.click();`

That is, it is possible.
But such a click only works if the menu item is expanded, of course.

**Reply by dvesh3** [reply]
you can use `pimcore.helpers.openObject` and pass id https://github.com/pimcore/pimcore/blob/10.x/bundles/AdminBundle/Resources/public/js/pimcore/helpers.js#L153

**Reply by esisl** [reply]
I apologize for the stupidity, but:
`pimcore.helpers.openObject(129, 'object');`

Uncaught TypeError: Cannot read property 'openObject' of undefined

**Reply by esisl** [reply]
Sorry!

`window.top.pimcore.helpers.openObject(129, 'object');`

**Reply by ShaileshT-dot** [reply]
Probably you dont have the latest helpers.js file. can you compare the helpers.js file in your codebase and github one?

**Reply by esisl** [reply]
No, I was just referring to the wrong namespace, as I understand it.

Now everything works
